Top10Words/DataFromBook.cs

```diff
@@ -1,13 +1,9 @@
-using System.Collections.Generic;
-using System.IO;
-
 namespace Top10Words;
 
 public class DataFromBook
 {
-    private string _fileName;
+    private readonly string _fileName;
     public IEnumerable<string> BookText;
```

Comment:
Since BookText is public, it should be converted to property and make it impossible to modify from outside.